api/xapi.rb

```diff
@@ -0,0 +1,171 @@
+require 'sinatra/base'
+require 'sinatra/petroglyph'
+require 'trackler'
+require 'exercism'
+require "bugsnag"
```

Comment:
Aesthetic but did you want to keep it consistent and use single quotes instead of double?